manuscript.md

```diff
@@ -1,8 +1,33 @@
-# Meta
+---
+bibliography: [references.bib]
+---
 
 **BOX 1: Biological Examples**
 
-## What is the paper about?
+## Why should we predict and forecast species interaction networks?
+
+Networks of species interactions underpin our understanding of key ecological
+processes [@Pascual2006EcoNet; @Heleno2014EcoNet]. Although they have initially
+been used to describe the interactions *within* a community, our interest in the
+last decade has shifted towards understanding their structure and their
+variation over space [@Trojelsgaard2016EcoNet; @Baiser2019EcoRul], and
+established network ecology as an important emerging component of biogeography
+and macroecology. But our understanding of network structure, and in particular
+across space, is limited by availability of data. Characterizing a network
+requires an exhaustive list of species interactions at the sampled location, and
+these data remain extremely scarce. This is because sampling ecological
+interactions is extremely difficult [@Jordano2016SamNet]; in turn, the
+difficulty of sampling interactions can lead to biases in our estimate of the
+sampling of network structure [@deAguiar2019RevBia]. As many ecological systems,
```

Comment:
In:
>in turn, the difficulty of sampling interactions can lead to biases in our estimate of the sampling of network structure

Are you discussing biases in our estimates of network structure or biases in our sampling of ecological networks? I think this sentence could be slightly clearer 😄

Reply:
Good call, fixed in the next commit.